Sources/Core/Sources/Render/Shader/ChunkShaders.metal

```diff
@@ -89,3 +89,38 @@ fragment float4 chunkFragmentShader(RasterizerData in [[stage_in]],
 
   return color;
 }
+
+
+constant float2 quadVertices[] = {
```

Comment:
Move these functions to a new file like `ScreenShaders.metal` or something

Reply:
Moved to `something.metal` 🫡